ui-admin/src/study/drag-drop-utils.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { useDraggable, useDroppable } from '@dnd-kit/core'
+import { CSS } from '@dnd-kit/utilities'
+
+/**
+ *
+ */
+export function Draggable(props: {id: string; item: object, children: React.ReactNode}) {
```

Comment:
are these components ever used? I was trying to figure out how they worked and couldn't find any usages.

Reply:
ah, you're right, these were my first attempt at utility helpers, but didn't end up being as good as the table helpers